### PR TITLE
doc: ai update conftest.py

### PR DESCRIPTION
**Pull request by ganwell**

### CODE REVIEW

File: pyaptly/conftest.py [hunk 1]
```diff
@@ -75,22 +76,27 @@ def environment(debug_mode):
...
+    specific datetime. For example:
 
-    ```python
-    @pytest.mark.parametrize("freeze", ["2012-10-10 10:10:10"], indirect=True)
-    def test_snapshot_create_basic(environment, config, freeze):
-    ...
-    ```
+    ```python @pytest.mark.parametrize("freeze", ["2012-10-10 10:10:10"],
+    indirect=True) def test_snapshot_create_basic(environment, config, freeze):
+    ... ```
```
Comment (winged) [inline pyaptly/conftest.py:99]:
```suggestion
    ```python
    @pytest.mark.parametrize("freeze", ["2012-10-10 10:10:10"], indirect=True)
    def test_snapshot_create_basic(environment, config, freeze):
        ...
    ```
```

File: pyaptly/conftest.py [hunk 2]
```diff
@@ -102,13 +108,16 @@ def test_snapshot_create_basic(environment, config, freeze):
...
 
-    Can be configured with:
+    This fixture can be configured with a specific configuration file. For
+    example, to use "mirror-extra.toml" as the configuration, annotate a test
+    function like this:
 
     ```python
-    @pytest.mark.parametrize("config", ["mirror-extra.toml"], indirect=True)
-    def test_mirror_create(environment, config, caplog):
+    @pytest.mark.parametrize("freeze", ["2012-10-10 10:10:10"], indirect=True)
```
Comment (winged) [inline pyaptly/conftest.py:119]:
```suggestion
    @pytest.mark.parametrize("config", ["mirror-extra.toml"], indirect=True)
```

File: pyaptly/conftest.py [hunk 3]
```diff
@@ -102,13 +108,16 @@ def test_snapshot_create_basic(environment, config, freeze):
...
+    def test_snapshot_create_basic(environment, config, freeze):
```
Comment (winged) [inline pyaptly/conftest.py:120]:
```suggestion
    def test_mirror_create(environment, config, caplog):
```

**Review by winged — CHANGES_REQUESTED** [review]
Some rewriting went the wrong way with code examples. Otherwise LGTM

File: pyaptly/conftest.py (hunk 3 above)
Comment (ganwell) [reply pyaptly/conftest.py:120]: Sorry. I had to fix that 100 times. And the last time I forgot!

**Comment by winged** [comment]
If needed we can still look at this. Closing. (JLF)

**Pull request closed without merging**